packages/tradershub/src/features/cfd/modals/TradeModal/TradeModal.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect } from 'react';
+import QRCode from 'qrcode.react';
+import { Provider } from '@deriv/library';
+import { Text, useBreakpoint } from '@deriv/quill-design';
+import { Modal } from '../../../../components/Modal';
+import { THooks, TMarketTypes, TPlatforms } from '../../../../types';
+import { AppToIconMapper, CFDPlatforms, LinksMapper, PlatformDetails, TAppLinks } from '../../constants';
+import { MT5TradeScreen } from '../../screens/MT5TradeScreen';
+
+type TTradeModalProps = {
+    account?: THooks.CtraderAccountsList | THooks.DxtradeAccountsList | THooks.MT5AccountsList;
+    marketType?: TMarketTypes.All;
+    platform: TPlatforms.All;
+};
+
+const TradeModal = ({ account, marketType, platform }: TTradeModalProps) => {
+    const { isDesktop } = useBreakpoint();
+    const { setCfdState } = Provider.useCFDContext();
+
+    useEffect(() => {
+        setCfdState('marketType', marketType);
+        setCfdState('platform', platform);
+        if (platform === CFDPlatforms.MT5) setCfdState('accountId', (account as THooks.MT5AccountsList)?.loginid);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
+
+    const appOrder = ['ios', 'android', 'huawei'];
+
+    return (
+        <Modal>
+            <Modal.Header title='Trade' />
+            <Modal.Content>
+                <MT5TradeScreen account={account} />
+            </Modal.Content>
+            <Modal.Footer>
+                <div className='pt-50 min-h-[19rem] flex justify-center items-center flex-col h-fit w-full gap-800'>
+                    <Text align='center' size='sm' weight='bold'>
+                        Download {PlatformDetails[platform].title} on your phone to trade with the{' '}
+                        {PlatformDetails[platform].title} account
+                    </Text>
+                    <div className='flex gap-800'>
+                        <div className='flex flex-col justify-center gap-400'>
+                            {appOrder.map(app => {
+                                const AppsLinkMapper = LinksMapper[platform][app as keyof TAppLinks];
+                                if (AppsLinkMapper) {
+                                    const AppIcon = AppToIconMapper[app];
+                                    const appLink = AppsLinkMapper;
+                                    return (
+                                        <AppIcon
+                                            className='w-[13.7rem] h-[4rem] cursor-pointer'
```

Comment:
Pls use px for custom properties. Michio told me that our tailwind does not bode well with rem

Reply:
To follow up above:
In deriv.app `rem` it's configured where `1rem` = `10px`. However, here we haven't set the configuration just yet and some of the tailwind classes might not satisfy our use case hence it is better to use `px` as of now.